File: wurster_lab01/wurster_lab01.swift
func main() {
    print("\nenter three integers")
    let nums = [Int(Double(readLine() ?? "0") ?? 0),
                Int(Double(readLine() ?? "0") ?? 0),
                Int(Double(readLine() ?? "0") ?? 0)]
    let mean = getMean(sum: Double(nums.reduce(0, {sum, next in sum + next})), count: nums.count)
    print("The mean of \(nums) was " + String(mean) + ". \nenter another integer")
    let newNum = Int(Double(readLine() ?? "0") ?? 0)
    let newMean = getMean(sum: (mean + Double(newNum)), count: 2)
    print("The mean of " + String(mean) + " and " + String(newNum) + " was " + String(newMean) + ".\n")
}

func getMean(sum: Double, count: Int) -> Double {
    return (sum / Double(count))
}

main()

File: wurster_lab01/wurster_lab01.py
def main():
	print('\nenter three integers')
	nums = [int(input('num1: ')),
			int(input('num2: ')),
			int(input('num3: '))]
	mean = getMean(sum(nums), len(nums))
	print('The mean of those three was ' + str(mean) + '.\nenter another integer')
	newNum = int(input('num4: '))
	newMean = getMean(mean + newNum, 2)
	print('The mean of ' + str(mean) + ' and ' + str(newNum) + ' was ' + str(newMean) + '.\n')


def getMean(sum, count):
	return float(sum) / count
	

main()